src/QuadForm/ClosestVector.jl

```diff
@@ -69,20 +59,26 @@ function pojective_quadratic_triple(Q::MatrixElem, L::MatrixElem, c::RingElement
         push!(pp1, p1)
         push!(LL1, L1)
         push!(r1, Q[n+1-i, n+1-i])
-        push!(m1, L[n+1-i,1:1])
+        push!(m1, L[n+1-i,1:1][1])
         pL =[Lqqt, pp1, LL1, r1, m1]
     end
     #------------------------------------------------
+    c1 = c; pQT = []; ppQT = []; pLL = []; 
+    ppLL = []; cc1 = []; Lqt = [];
```

Comment:
Note that these arrays are quite inefficient since they are of type `Vector{any}`.

Reply:
I'll keep it in mind. Will improve the code gradually:)